pkg/gameservers/health.go

```diff
@@ -93,8 +93,7 @@ func NewHealthController(health healthcheck.Handler,
 			// Could be a DeletedFinalStateUnknown, in which case, just ignore it
 			pod, ok := obj.(*corev1.Pod)
 			if ok && isGameServerPod(pod) {
-				owner := metav1.GetControllerOf(pod)
-				hc.workerqueue.Enqueue(cache.ExplicitKey(pod.ObjectMeta.Namespace + "/" + owner.Name))
+				hc.workerqueue.Enqueue(pod)
```

Comment:
Me also doing another cheeky cleanup. The previous version was before pods+gameservers didn't have the same name, so we had to build the GameServer name from the pod.

We don't have to do that anymore, since worker.Enqueue() will by default build exactly the same key from the pod's namespace and name. So it's actually more concise, and less processing - but the functionality is identical.

Reply:
It sounds like lines 88-89 should also be updated.

Reply:
Nice catch!